### PR TITLE
You can now enslave IPCs.

### PR DESCRIPTION
**Pull request by FlattestGuitar**
:cl: FlattestGuitar
rscadd: You can now enslave IPCs by putting a coin into their breadslot.
/:cl:

You can now enslave an IPC by putting a coin inside their breadslot, just like nature intended.

### CODE REVIEW

**Comment by Ty-Omaha** [comment]
Where's the leashes????

**Comment by Wheatley29** [comment]
What

**Comment by Dumbdumn5** [comment]
http://niceme.me/

**Comment by Ty-Omaha** [comment]
http://nicememe.website/

**Comment by Ar3nn** [comment]
You can't be serious. This just gives miner traitors free mindslaves. Mine for ten minutes, all the coins you could fucking want.

**Comment by Wheatley29** [comment]
It is a joke

**Comment by FlattestGuitar** [comment]
@Ar3nn, mindslaves are already very expensive and underused and IPCs need more downsides to them.

We've talked this feature over and we believe it is in the best interest of the server to add it.

**Comment by Wheatley29** [comment]
On a similar notion, get ready for the mindslave vulpkanins with doggy biscuits

**Comment by Ar3nn** [comment]
So the answer to mindslaves being expensive and underused is to make them redundant by saying "go to the asteroid and get a bunch of free versions of that."

**Comment by IcyV** [comment]
Most sensible PR ever. Insta-merge material.

**Comment by Wheatley29** [comment]
@Ar3nn , As an IPC player, I can see this being very beneficial. I for one am excited for this to merge

**Comment by Dumbdumn5** [comment]
I am in full support of this, as IPCs are an incredibly powerful and tanky race, which are incredibly hard to kill and able to repair all damage to themselves in a single second. Also, they don't need to eat and they're immune to oxygen damage. OXYGEN DAMAGE!  AND THE TOXINS! RADIATION DOESN'T HURT THEM! Do you know how OP that is? Then they have a clique language that makes it so they can sit in a corner and act all computers n' shit. This is completely necessary to my religious and political beliefs, merge immediately.

**Comment by Ar3nn** [comment]
In the mindset that there have been a lot of things added over time, I am considering doing a PR to tweak TC costs for some items. Knowing this to be a concern, I might look into it. But this is not a fix to the problem. And @Wheatley29 I am also an IPC player, but this is basically saying fuck you IPC you're always antag because the antag will literally never not have a reason to convert you.

**Comment by Ar3nn** [comment]
@Dumbdumn5 One of the explicit nerfs to IPC is they take more damage from all sources they are vulnerable to by a considerable factor. If you're depending entirely on oxy damage to kill the entire station for your hijack, reconsider your plan, mate. Also, I highly doubt even the most robust IPC can repair themselves efficiently in combat, and while they're distracted, just hit them some more.

**Comment by Wheatley29** [comment]
@Ar3nn an IPC can easily be taken down with an Ion rifle, they should be used as a henchman as they are  very easy to dispose of. And there are reasons not to convert IPC's seeing as how they will waste any gear you give them or fail any objective if security is smart enough to use that ion rifle

**Comment by FlattestGuitar** [comment]
That's true. Another important factor is considering that most IPC players are fresh arrivals to the server and will most likely prove to be more of a liability than an asset when mindslaved.

**Comment by Ar3nn** [comment]
Now I feel targeted and insulted.

**Comment by Dumbdumn5** [comment]
What about the fact that IPCs can LITERALLY ignore any viruses, making them practically unkillable as a traitor virologist! And phoron does NOTHING to them!

**Comment by Wheatley29** [comment]
@Ar3nn , why?

**Comment by Ar3nn** [comment]
The fuck is a breadslot anyway I've literally never heard of it. And viruses are and have always been a shit way to assassinate. No matter how obscure the cure is, there's always a chance it gets quarantined before your target contracts.

**Comment by IcyV** [comment]
I've recently uncovered a horrible secret. IPCs control the economy. We need to take them down or we all go down!

**Comment by Dumbdumn5** [comment]
And then the injection, I can't sleepy pen IPCs! Ether does NOTHING to them! They're INCREDIBLY hard to take down quietly!

**Comment by Wheatley29** [comment]
@Ar3nn 
The below image is a breadslot
image

**Comment by Dumbdumn5** [comment]
What about the shaving block?

**Comment by Ar3nn** [comment]
The shaving block is only because a couple of barber mains would literally go out of their way to shave IPCs and iirc it wasn't properly coded so it made them unrecognizable, which required surgery to fix. Over a 2 second joke the barber pulled repeatedly on them. Clown slip, shave, they need 5-10 minutes of their round ate up by surgery. Is this fun? Also, that's a toaster.

**Comment by Ar3nn** [comment]
Also, yes, let my data port process this organic chemical substanc- oh wait.

**Comment by Dumbdumn5** [comment]
Shaving Block

**Comment by Dumbdumn5** [comment]
THAT is a SHAVING BLOCK

**Comment by Ar3nn** [comment]
That's a fucking cheese grater.

**Comment by Wheatley29** [comment]
We should also make IPC's need to "trim" their wires, the attached device could be used for such
image

**Comment by Dumbdumn5** [comment]
A circular saw?

**Comment by Ar3nn** [comment]
Trimming wires can cause serious electrical issues, even if you don't outright sever the connector. Also, touching metal to active conductive elements. Fuck that.

**Comment by Wheatley29** [comment]
Do notice the rubber handle

**Comment by FlattestGuitar** [comment]
I agree with @Ar3nn, this is an extremely silly idea.

Wire trimming should ONLY be done by trained professionals and while we try to make our game as playable as possible, it's important that we teach out audience proper cable trimming etiquette.

**Comment by Ar3nn** [comment]
Rubber dampens the effect, but if any of it reached the IPC's hand, the circuit would be made and fuck a lot of shit up.

**Comment by Wheatley29** [comment]
To add more immersion to IPCs, it should be mandatory for them to charge with a special adapter they spawn with, here is a concept image

image

**Comment by Dumbdumn5** [comment]
IPCs aren't even remotely balanced, they should have to carry around this water cooler to cool their machine parts! image of a desk fan

**Comment by tristan1333** [comment]
This a mighty fine PR friend MIGHTY fine JUST fine AWAAA fine PR friend, I'm thinking this will be advancing the codebase MIGHTY far friend GOING far we are right here we will MERGE right here right here MERGE

**Comment by Lady-Luck** [comment]
>http://nicememe.website/

http://nicememewebsite.website/

**Comment by FlattestGuitar** [comment]
http://nicememewebsitewebsitewebsitewebsitewebsite.website/

**Comment by Dumbdumn5** [comment]
> http://nicememewebsitewebsitewebsitewebsitewebsite.website/
https://www.youtube.com/watch?v=eCiFO7qV54E

**Comment by DaveTheHeadcrab** [comment]
JESUS FUCK MORE MINING POWERCREEP USE THE MINT TO GET INFINITE MINDSLAVES WHAT THE FUCKING SHIT IS THE MATTER WITH YOU PEOPLE?!

File: code/modules/mining/coins.dm [hunk 1]
```diff
@@ -149,4 +149,15 @@
...
 								 "<span class='notice'>You flip [src]. It lands on [coinflip].</span>", \
-								 "<span class='notice'>You hear the clattering of loose change.</span>")
\ No newline at end of file
+								 "<span class='notice'>You hear the clattering of loose change.</span>")
+
+/obj/item/weapon/coin/attack(mob/M as mob, mob/user as mob)
+	if(!M.isSynthetic() || !ishuman(M) || M == user)
+		return ..()
+	else
+		to_chat(user, "You [pick("silently", "sensually", "frantically", "aggressively", "rapidly", "slowly", "clumsily", "spectacularly")] insert [src] into [M]'s breadslot, bending them permanently to your will. Beep boop.")
```
Comment (VampyrBytes) [inline code/modules/mining/coins.dm]: "You sensually insert [src] into [M]'s breadslot"
:blush:

**Comment by Spacemanspark** [comment]
Why did you wait to make this meme PR until after I had left for OSUT.
I demand answers @FlattestGuitar

**Comment by FlattestGuitar** [comment]
I wanted you to have a sensual surprise. : )

**Comment by Spacemanspark** [comment]
You're fired

**Comment by taukausanake** [comment]
Never thought I'd see this PR again. Thanks :)

**Pull request closed without merging**